sys/cloudformation/stack.yaml

```diff
@@ -342,10 +361,11 @@ Resources:
   # ECS TASK DEFINITION
   ecstask:
     Type: AWS::ECS::TaskDefinition
+    DeletionPolicy: Retain
     Properties:
       ExecutionRoleArn: !Ref ExecRoleArn
       ContainerDefinitions:
-      
+        # container phpfpm
```

Comment:
LOL .. I removed the comment in the last PR as the Name is now the first line and should be enough ... you don't agree I guess ;P

Reply:
fixed